### PR TITLE
Suggest page4

### PR DESCRIPTION
**Pull request by julianecasier**
New version of the suggestion page, adding the class and a selected input

### CODE REVIEW

**Review by Elfaram — DISMISSED** [review]

**Comment by LoicBrassart** [comment]
Plz don't assign me yourself (Luigi'll do it when it's time)

**Comment by Elfaram** [comment]
...i didn't do this...

**Review by LuigiMenez — DISMISSED** [review]
Ok for me.

File: frontend/src/assets/datasuggest.js [hunk 1]
```diff
@@ -14,53 +14,62 @@ export default [
...
     picture: imgThinking,
     alt: "smiley thinking",
+    query: "with_genres=99&with_genres=36& with_genres=10752",
   },
   {
     key: "angry",
     type: "vibe",
     picture: imgAngry,
     alt: "smiley angry",
+    query: "with_genres=80&with_genres=27& with_genres=9648&with_genres=53",
```
Comment (LoicBrassart) [inline frontend/src/assets/datasuggest.js]: - Never add spaces to an url
- I may be wrong but I think you overwrite the same variable (`with_genre`) over and over, it doesn't send the three categories. Have you tested it ?

File: frontend/src/assets/datasuggest.js [hunk 2]
```diff
@@ -14,53 +14,62 @@ export default [
...
     picture: imgYoung,
     alt: "young",
+    query: "release_date.lte=2010-01-01&release_date.gte=1990-01-01",
   },
   {
     key: "baby",
     type: "dateRelease",
     picture: imgBaby,
     alt: "baby",
+    query: "release_date.gte=2010-01-01&release_date=2010-01-01",
```
Comment (LoicBrassart) [inline frontend/src/assets/datasuggest.js:59]: "Send me movies more recent than 2010-01-01 *and* which went live on 2010-01-01" ? Have you tested it ?

File: frontend/src/components/QuestionSug/index.jsx [hunk 3]
```diff
@@ -1,66 +1,85 @@
...
+            .filter((smiley) => smiley.type.includes("vibe"))
+            .map((smiley) => (
+              <input
+                name={smiley.key}
+                type="image"
+                src={smiley.picture}
+                alt={smiley.alt}
+                onClick={(event) => {
+                  handleChoiceLinks(event);
+                }}
```
Comment (LoicBrassart) [inline frontend/src/components/QuestionSug/index.jsx:37]: Simplifiable: `onClick={handleChoiceLinks}`

File: frontend/src/components/QuestionSug/index.jsx [hunk 4]
```diff
@@ -1,66 +1,85 @@
...
+            .filter((smiley) => smiley.type.includes("dateRelease"))
+            .map((smiley) => (
+              <input
+                name={smiley.key}
+                type="image"
+                src={smiley.picture}
+                alt={smiley.alt}
+                onClick={(event) => {
+                  handleChoiceLinks(event);
+                }}
```
Comment (LoicBrassart) [inline frontend/src/components/QuestionSug/index.jsx:54]: Same

File: frontend/src/components/QuestionSug/index.jsx [hunk 5]
```diff
@@ -1,66 +1,85 @@
...
+            .filter((smiley) => smiley.type.includes("runtime"))
+            .map((smiley) => (
+              <input
+                name={smiley.key}
+                type="image"
+                src={smiley.picture}
+                alt={smiley.alt}
+                onClick={(event) => {
+                  handleChoiceLinks(event);
+                }}
```
Comment (LoicBrassart) [inline frontend/src/components/QuestionSug/index.jsx:71]: Same

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
Some major Evils to fix !

**Comment by julianecasier** [comment]
> ...i didn't do this...

I did... sorry, noted

**Review by Elfaram — APPROVED** [review]

**Pull request closed without merging**